developer-workspace/common/deploy-kube-prometheus.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-kubectl apply -f manifests/setup
+kubectl create -f manifests/setup
```

Comment:
@paulfantom - why do we treat the script differently than the recommendation in the docs?

Reply:
When I wrote it I didn't know which k8s version allowed for `--server-side` flag without going through feature flags. Thus I assumed it would be safer to use `create` especially if this script is always run on a fresh cluster.

After some docs checking I think it should be fine with `apply --server-side` instead of `create`.

Reply:
Not sure how many people use this script, but I always use it with the command `make deploy`.

Make deploy will download the latest version of kind, so I'd assume it is safe to use server-side apply as well :)